pq-crypto/s2n_pq.c

```diff
@@ -95,13 +95,21 @@ bool s2n_sikep434r2_asm_is_enabled() {
     return sikep434r2_asm_enabled;
 }
 
+bool s2n_pq_is_enabled() {
+#if defined(S2N_NO_PQ)
+    return false;
+#else
+    return !s2n_is_in_fips_mode();
```

Comment:
I was a little confused how you were stopping both PQ and FIPS from both being enabled until I found this line. I think you should call out this solution of doing the FIPS check inside the PQ check more prominently in the PR description.

Reply:
I updated the PR description with a suggestion of what order to review the files in. I think that will give future reviewers the best shot at having the correct context in mind when looking at the code changes.